### DISCUSSION TITLE
Different images for different PCs

### DISCUSSION THREAD

**Discussion by Mogaba** [ltsp/ltsp#792] [Q&A] [answered]
We have some very old PCs that don't work well with the recent Ubuntu versions so we have to use 18.04.
Recently we got some new PCs that don't work well with Ubuntu 18.04 so we need either 20.04 or 22.04.

I guess the best solution would be to configure LTSP server to provide images based on MAC, is it possible?

**Comment by alkisg** [comment, marked as answer]
Sure. Name the images x86_64.img and x86_32.img. The x86_64 one will be the default for all 64bit clients, and the x86_32 for the 32bit clients.
While to force 64bit client to use the 32bit image, put sections like this in ltsp.conf, and run ltsp ipxe afterwards:

```ini
[mac:address:of:old:pc]
DEFAULT_IMAGE=x86_32
```
